### PR TITLE
Add COS alert rules and update COS dashboard

### PR DESCRIPTION
**Pull request by jakobilobi**
Configuring the Polkadot dashboard + adding basic alert rules.

- [x] Add block production alert rule
- [x] Add datasource missing alert rule
- [x] Figure out what Loki alert rule to add, e.g. what log messages to monitor
- [x] Adapt dashboard to COS setup
- ~[ ] Explore adding a dashboard panel based on Loki data, e.g. log messages/hour or something like that~

### CODE REVIEW

File: src/alert_rules/loki/error-in-syslog.rule [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            Unit: {{ $labels.juju_unit }}
+            Instance: {{ $labels.instance }}
+            LABELS = {{ $labels }}
+      - alert: DEV__TargetOverTime
+        expr: count_over_time({filename="/var/log/syslog", juju_charm=~".*", juju_application="polkadot"} |= `target` [1m]) > 10
+        for: 5m
+        labels:
+          severity: error
+        annotations:
+          summary: "Excessive amount of 'target' found"
```
Comment (erik78se) [inline src/alert_rules/loki/error-in-syslog.rule:23]: What does target then indicate? Best to provide information about that for someone that needs to understand the implications.

Comment (jakobilobi) [reply src/alert_rules/loki/error-in-syslog.rule:23]: As I wrote in the description of the PR these are just placeholders. E.g. perhaps we want to alert if "something" shows up in the log more than 10 times every minute, then we would do that like this.

**Comment by erik78se** [comment]
Ah, I understand.



Den fre 7 juli 2023 19:51Jakob ***@***.***> skrev:

**Comment by erik78se** [comment]
Makes sense.

We should feedback on this to parytech on the excessive error messages

Den tis 11 juli 2023 11:44Jakob ***@***.***> skrev:

**Comment by Maharacha** [comment]
> This is now ready for review:
> 
> * Added alert rules for block production and missing prometheus endpoint, e.g. prometheus down.
> * Updated [dashboard](http://192.168.110.10/cos-grafana/d/dwellir-polkadot-generic/polkadot-dotsama-ecosystem-dashboard?orgId=1): irrelevant dashboard variables are now hidden, panels show data from selected units only (before was from all of the model).
> 
> Comments:
> 
> * There are a lot of default alerts, [see list](http://192.168.110.10/cos-grafana/alerting/list) where our own ones are only 4 out of 40. Haven't done anything about them as I suspect most are actually useful, let's talk about them later on if needed.
> * Haven't dug into expanding the dashboard with a panel based on Loki data yet, I think we can leave that until we see an opportunity or feel like something is missing.

Interesting with the Watchdog alert rule.

**Pull request merged**